internal/controller/postgrescluster/volumes.go

```diff
@@ -652,6 +654,7 @@ func (r *Reconciler) reconcileMoveRepoDir(ctx context.Context,
 					Image:        config.PGBackRestContainerImage(cluster),
 					Name:         naming.ContainerJobMovePGBackRestRepoDir,
 					VolumeMounts: []corev1.VolumeMount{pgbackrest.RepoVolumeMount()},
+					Resources:    cluster.Spec.Backups.PGBackRest.RepoHost.Resources,
```

Comment:
:wrench:  Similar to the above, but I believe it's likely there are cases where `RepoHost` will be `nil`.